libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/addAttributeFilter/AttributesDropdown.tsx

```diff
@@ -125,10 +135,39 @@ export function AttributesDropdown({
         "add-attribute-filter-dropdown",
     );
 
-    const filteredMeasures = searchQuery
+    const filteredAttributes = searchQuery
         ? attributes.filter((a) => a.attribute.title.toLowerCase().includes(searchQuery.toLowerCase()))
         : attributes;
 
+    const offerDateFilters = enableMultipleDateFilters && supportsMultipleDateFilters;
+
+    const filteredDateDatasets = searchQuery
+        ? dateDatasets.filter((ds) => ds.dataSet.title.toLowerCase().includes(searchQuery.toLowerCase()))
+        : dateDatasets;
+
+    const [selectedTabId, setSelectedTabId] = useState(
+        attributes.length || !offerDateFilters ? "attributes" : "dateDatasets",
```

Comment:
Super minor, but you don't need negation if you switch the branches ;)

Reply:
fixed